sentry/src/main/java/io/sentry/MainEventProcessor.java

```diff
@@ -66,23 +67,35 @@ public MainEventProcessor(final @NotNull SentryOptions options) {
   }
 
   private void setDebugMeta(final @NotNull SentryBaseEvent event) {
+    final @NotNull List<DebugImage> debugImages = new CopyOnWriteArrayList<>();
```

Comment:
nit: this could be `ArrayList`, not sure if it ever will be modified from different threads